src/coreclr/tools/Common/TypeSystem/Ecma/EcmaModule.cs

```diff
@@ -320,12 +320,12 @@ public sealed override MetadataType GetType(string nameSpace, string name, NotFo
                             }
                             if (implementation is ModuleDesc moduleDesc)
                             {
-                                return moduleDesc.GetType(nameSpace, name);
+                                return moduleDesc.GetType(nameSpace, name, notFoundBehavior);
```

Comment:
This is a bugfix - I noticed it while reviewing the pull request from earlier this week.